tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -985,11 +998,177 @@ public void SetDefaultExtensions(string recordingId = null)
+                    new BodyKeySanitizer("$..applicationSecret"),
+                    new BodyKeySanitizer("$..apiKey"),
+                    new HeaderRegexSanitizer("api-key"),
+                    new BodyKeySanitizer("$..connectionString"),
+                    new GeneralRegexSanitizer(regex: "Accesskey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    new GeneralRegexSanitizer(regex: "Secret=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    new HeaderRegexSanitizer("x-ms-encryption-key"),
+                    new BodyKeySanitizer("$..sshPassword"),
+                    new BodyKeySanitizer("$..aliasSecondaryConnectionString"),
+                    new BodyKeySanitizer("$..primaryKey"),
+                    new BodyKeySanitizer("$..secondaryKey"),
+                    new BodyKeySanitizer("$..adminPassword.value"),
+                    new BodyKeySanitizer("$..administratorLoginPassword"),
+                    new BodyKeySanitizer("$..accessToken"),
+                    new BodyKeySanitizer("$..runAsPassword"),
+                    new BodyKeySanitizer("$..adminPassword"),
+                    new BodyKeySanitizer("$..accessSAS"),
+                    // "token": "sv=...ss=...srt=...se=...sp=...sig=" request body covered by 1052
+                    new BodyKeySanitizer("$..WEBSITE_AUTH_ENCRYPTION_KEY"),
+                    new BodyKeySanitizer("$..decryptionKey"),
+                    new HeaderRegexSanitizer("ServiceBusDlqSupplementaryAuthorization", regex: "(?:(sv|sig|se|srt|ss|sp)=)(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    new HeaderRegexSanitizer("ServiceBusSupplementaryAuthorization", regex: "(?:(sv|sig|se|srt|ss|sp)=)(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    // "RequestBody": "client_id=...grant_type=...client_info=...client_secret=…scope=...", covered by 1036/1037
                     new BodyKeySanitizer("$..access_token"),
-                    new BodyKeySanitizer("$..refresh_token")
+                    new BodyKeySanitizer("$..AccessToken"),
+                    new BodyRegexSanitizer(regex: "(client_id=)(?<cid>[^&]+)", groupForReplace: "cid"),
+                    new BodyRegexSanitizer(regex: "client_secret=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    new BodyRegexSanitizer(regex: "client_assertion=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    // new BodyKeySanitizer("$..targetModelLocation"), disabled, not a secret?
+                    new BodyKeySanitizer("$..targetResourceId"),
+                    new BodyKeySanitizer("$..urlSource"),
+                    new BodyKeySanitizer("$..azureBlobSource.containerUrl"),
+                    new BodyKeySanitizer("$..source"),
+                    // new BodyKeySanitizer("$..resourceLocation"), disabled, not a secret?
+                    new HeaderRegexSanitizer("Location"),
```

Comment:
The replacement value should be a valid URL to avoid breaking polling scenarios

Reply:
Updated to `example.com`